libs/platform/ebpf_platform.h

```diff
@@ -415,7 +415,10 @@ extern "C"
      * @retval EBPF_NOT_FOUND Key not found in hash table.
      */
     ebpf_result_t
-    ebpf_hash_table_find(ebpf_hash_table_t* hash_table, const uint8_t* key, uint8_t** value);
+    ebpf_hash_table_find(
+        _In_ ebpf_hash_table_t* hash_table,
+        _In_ _Readable_bytes_(hash_table->key_size) const uint8_t* key,
+        _Outptr_ _Writable_bytes_(hash_table->value_size) uint8_t** value);
```

Comment:
_Outptr_opt_result_maybenull_ ?

Reply:
Not sure exactly what the correct annotation should be. 

Requirements for these set of functions is:
Out can't be null.
On success, points to a buffer of hash_table->value_size bytes.

Reply:
Outptr is correct then (see https://docs.microsoft.com/en-us/cpp/code-quality/annotating-function-parameters-and-return-values?view=msvc-160#output-pointer-parameters).